portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,22 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> test;
 
+  @Override
+  public void init() { 
+    test = new ArrayList<>();
```

Comment:
Logging statements are usually fine to check in for debugging, but actual tests should either be in a test file or left out of the commits.

Reply:
changed